client/my-sites/plans/ecommerce-trial/entrepreneur-plan/entrepreneur-plan.jsx

```diff
@@ -0,0 +1,158 @@
+		plan.montlyPrice = getPlanRawPrice( state, rawPlans[ plan.slug ].getProductId(), true ) || 0;
+		plan.discount = Math.floor( ( 1 - plan.montlyPrice / baseMontlyPrice ) * 100 );
+		plan.discountText = translate( '%(discount)d%% off', {
+			args: { discount: plan.discount },
+			comment: 'Discount percentage',
+		} );
+		switch ( key ) {
+			case 'PLAN_ECOMMERCE':
+				plan.subText = translate( 'per month, %(rawPrice)s billed annually, excl. taxes', {
+					args: { rawPrice: plan.price },
+					comment: 'Excl. Taxes is short for excluding taxes',
+				} );
+				break;
+			case 'PLAN_ECOMMERCE_2_YEARS':
+				plan.subText = translate( 'per month, %(rawPrice)s billed every two years, excl. taxes', {
+					args: { rawPrice: plan.price },
+					comment: 'Excl. Taxes is short for excluding taxes',
+				} );
+				break;
+			case 'PLAN_ECOMMERCE_3_YEARS':
+				plan.subText = translate( 'per month, %(rawPrice)s billed every three years, excl. taxes', {
+					args: { rawPrice: plan.price },
+					comment: 'Excl. Taxes is short for excluding taxes',
+				} );
+				break;
+		}
+	} );
+
+	return planPrices;
+};
+
+export function EntrepreneurPlan() {
+	const translate = useTranslate();
+	const selectedSite = useSelector( getSelectedSite );
+	const plans = useEntrepreneurPlanPrices();
+	const currencyCode = useSelector( getCurrentUserCurrencyCode );
+	const [ selectedInterval, setSelectedInterval ] = useState( 'PLAN_ECOMMERCE' );
+	const selectedPlan = plans[ selectedInterval ];
+
+	const selectControlOptions = Object.keys( plans ).map( ( key ) => {
+		const plan = plans[ key ];
+		return {
+			key,
+			name: (
+				<div>
+					{ plan.term } <span>{ plan.discountText }</span>
+				</div>
+			),
+		};
+	} );
+
+	const upgradeClick = () => {
+		const checkoutUrl = getTrialCheckoutUrl( {
+			productSlug: selectedPlan.slug,
+			siteSlug: selectedSite.slug,
+		} );
+
+		page.redirect( checkoutUrl );
+	};
+
+	return (
+		<>
+			<h2 className="entrepreneur-trial-plan__section-title">
+				{ translate( "What's included in your free trial" ) }
+			</h2>
+			<div className="entrepreneur-trial-plan__included-wrapper">
+				<EcommerceTrialIncluded displayAll={ true } />
+			</div>
+			<div className="plan-heading">
+				<h2 className="entrepreneur-trial-plan__section-title">
+					{ translate( 'Keep all the features for your site' ) }
+				</h2>
+				<CustomSelectControl
+					options={ selectControlOptions }
+					className="period-select"
+					hideLabelFromVision
+					onChange={ ( { selectedItem: { key } } ) => {
+						setSelectedInterval( key );
+					} }
+				/>
+			</div>
+			<Card>
+				<div className="plan-wrapper">
+					<div className="plan-description">
+						<h3 className="entrepreneur-trial-plan__plan-title">{ translate( 'Entrepreneur' ) }</h3>
+						<p className="card-text">
+							{ translate(
+								"Continue enjoying the full benefits of Entrepreneur plan, simply add your pay method and maximize your store's potential."
```

Comment:
This is probably coming from the design, but just wanted to check: don't we rather use `payment plan` here? (the same as we currently have in the CTA)

```suggestion
								"Continue enjoying the full benefits of Entrepreneur plan, simply add your payment method and maximize your store's potential."
```

![Markup on 2024-04-23 at 16:53:06](https://github.com/Automattic/wp-calypso/assets/25105483/ad796e16-acf3-4c34-a486-b29ad6f97912)

Reply:
Actually following up on this, we may consider changing the CTA copy and the description altogether. What we are asking the user is to actually purchase a full plan, rather than to enter a payment method only. 
Related: p1713898784084409/1713529322.545209-slack-C06RQ4EQ7V2

Reply:
However, this should not be PR blocking. We can merge in before deciding on the copies. Perhaps we can just refrain from submitting those for translation.

Reply:
Thank you, I addressed it. And kept the button text out of the translate context.

Reply:
The copy is being addressed in https://github.com/Automattic/wp-calypso/pull/89878.